.github/workflows/test-install.yml

```diff
@@ -72,6 +72,14 @@ jobs:
       run:
         python -c "import aiida"
 
+    - name: Send Slack notification
+      if: ${{ failure() && github.event_name == 'schedule' }}
```

Comment:
after #4689 is merged, I think this will never trigger, because the `continue-on-error` has been removed

Reply:
The `continue-on-error` field was previously added to ensure that the overall workflow continues even if there are issues with the pip-2020-resolver. To the best of my knowledge this should not affect the execution of those steps to run on failure.

Reply:
oh yeh I think you are correct